libpod/volume.go

```diff
@@ -49,6 +49,8 @@ type VolumeConfig struct {
 	UID int `json:"uid"`
 	// GID the volume will be created as.
 	GID int `json:"gid"`
+	// Size maximum of the volume.
+	Size int64 `json:"gid"`
```

Comment:
Should we store this as uint64 given that `SetQuota()` accepts uint64?
Also the json field name is wrong `s/gid/size/`.